pycalphad/model.py

```diff
@@ -202,7 +202,10 @@ def __init__(self, dbe, comps, phase_name, parameters=None):
         self.build_phase(dbe)
 
         for name, value in self.models.items():
-            self.models[name] = self.symbol_replace(value, symbols)
+            # XXX: xreplace hack because SymEngine seems to let Symbols slip in somehow
+            self.models[name] = self.symbol_replace(value, symbols).xreplace({Symbol('T'): v.T,
+                                                                              Symbol('P'): v.P,
+                                                                              Symbol('R'): v.R})
```

Comment:
was this due to the issue with sublattice Symbol in variables (that should be fixed now)?

Reply:
That code will probably be deleted when symengine 0.9.0 releases

Reply:
Still seem to get a lot of test failures on symengine 0.9.0 with the xreplace hack removed

Reply:
I moved the dictionary into the variables module. I think this xreplace code is essential because symbol replacement doesn't happen anywhere in the database.